docs/notebooks/longrangecnot/LongRangeCNOTgates.ipynb

```diff
@@ -0,0 +1,2139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ba4719fa",
+   "metadata": {
+    "heading_collapsed": true,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Efficient Long-Range Entanglement using Dynamic Circuits\n",
+    "*Elisa Bäumer, Vinay Tripathi, Derek S. Wang, Patrick Rall, Edward H. Chen, Swarnadeep Majumder, Alireza Seif, Zlatko K. Minev*\\\n",
+    "**IBM Quantum**\n",
+    "\n",
+    "**Reference:** https://arxiv.org/abs/2308.13065\n",
+    "\n",
+    "**Abstract:** Quantum simulation traditionally relies on unitary dynamics, inherently imposing efficiency constraints on\n",
+    "the generation of intricate entangled states. In principle, these limitations can be superseded by non-unitary, dynamic circuits. These circuits exploit measurements alongside conditional feed-forward operations, providing a promising approach for long-range entangling gates, higher effective connectivity of near-term hardware, and more efficient state preparations. Here, we explore the utility of shallow dynamic circuits for creating longrange entanglement on large-scale quantum devices. Specifically, we study two tasks: CNOT gate teleportation between up to 101 qubits by feeding forward 99 mid-circuit measurement outcomes, and the preparation of Greenberger–Horne–Zeilinger (GHZ) states with genuine entanglement. In the former, we observe that dynamic circuits can outperform their unitary counterparts. In the latter, by tallying instructions of compiled quantum circuits, we provide an error budget detailing the obstacles that must be addressed to unlock the full potential of dynamic circuits. Looking forward, we expect dynamic circuits to be useful for generating long-range entanglement in the near term on large-scale quantum devices.\n",
```

Comment:
I'm a little bit conflicted about this part, let's see what other people also think. On one hand, I like the explicit reference (with abstract and authors), on the other hand it gives the impression that this is going to be the code of the paper while we actually cover only a part of the paper topics

Reply:
I changed this to use a different introduction which should be better.